keimed_auto_wave_picking/views/stock_move_line_views.xml

```diff
@@ -10,18 +10,20 @@
                 <field name="state" column_invisible="True" />
                 <field name="company_id" column_invisible="True" />
                 <field name="keimed_wave_id" column_invisible="True" />
-                <field name="product_id" context="{'default_detailed_type': 'product'}" required="1" readonly="1" />
-                <field name="lot_id" groups="stock.group_production_lot" readonly="tracking not in ['lot', 'serial'] or picked" />
-                <field name="location_id" readonly="picked" context="{'display_only_current_location_name': True}" />
-                <field name="location_dest_id" readonly="picked" />
+                <field name="product_id" context="{'default_detailed_type': 'product'}" required="1" readonly="context.get('operation_type') in ['detailed_operation', 'operation']" />
```

Comment:
Can't we use readonly="1"
insted of readonly="context.get('operation_type') in ['detailed_operation', 'operation']" ?